mkdocs.yml

```diff
@@ -24,7 +24,17 @@ theme:
     - navigation.tabs
     - navigation.tabs.sticky
     - toc.follow
-    - navigation.expand
```

Comment:
Ah ok, this is causing the nav to collapse when you move from something under Code to Get an API key (see comment in feedback doc). Need to add back `navigation.expand`